jre_emul/android/platform/libcore/ojluni/src/main/java/java/io/PrintWriter.java

```diff
@@ -901,7 +901,7 @@ public PrintWriter format(String format, Object ... args) {
                 ensureOpen();
                 if ((formatter == null)
                     || (formatter.locale() != Locale.getDefault()))
-                    formatter = new Formatter(WeakProxy.forObject(this));
+                    formatter = new Formatter(this);
```

Comment:
Please revert, as `new Formatter(this)` causes a reference cycle which WeakProxy avoids.